src/algorithms/admm.h

```diff
@@ -22,22 +22,22 @@ Results Family::fitADMM(const T& x, const mat& y, vec lambda, double rho){
   mat u(z);
   mat beta(z);
   mat beta_prev(z);
-  
-  // Ignore for now
+
   std::vector<double> primals;
   std::vector<double> duals;
   std::vector<double> time;
 
+  wall_clock timer;
+  if (diagnostics)
+      timer.tic();
 
-  uword passes=0;
   double alpha = 1.5;
-  
-  while(passes<max_passes){
+  uword passes=0;
```

Comment:
add space between relational operators (such as =)

```suggestion
  uword passes = 0;
```